dao/src/main/java/se/fortnox/reactivewizard/db/query/parts/CollectionOptionsQueryPart.java

```diff
@@ -17,6 +21,7 @@ public class CollectionOptionsQueryPart implements QueryPart {
     private final        int    collectionOptionsArgIndex;
     private final        Query  queryAnnotation;
     private final        boolean isMono;
+    private static final Pattern SORT_BY_PART = Pattern.compile("(?<sortby>[a-zåäö0-9_]+)(\\s+(?<order>asc|desc))?");
```

Comment:
Do we need åäö?

Reply:
I don't know. We are a swedish company...

Reply:
I want to think that åäö is not being used even though Fortnox is a swedish company :). Regardless, it is probably ok to leave it as is.